kouta-backend/src/main/scala/fi/oph/kouta/domain/oppilaitos.scala

```diff
@@ -364,6 +364,7 @@ case class OppilaitosMetadata(tietoaOpiskelusta: Seq[TietoaOpiskelusta] = Seq(),
                               ) extends ValidatableSubEntity {
   override def validate(tila: Julkaisutila, kielivalinta: Seq[Kieli], path: String): IsValid = and(
     validateIfNonEmpty[TietoaOpiskelusta](tietoaOpiskelusta, s"$path.tietoaOpiskelusta", _.validate(tila, kielivalinta, _)),
+    validateIfJulkaistu(tila, assertNotOptional(wwwSivu, s"$path.wwwSivu")),
```

Comment:
Tämän voisi yhdistää alempana olevaan `validateIfJulkaistu`-kutsuun `and()`:lla, kuten OppilaitoksenOsaMetadata:ssa